nf_core/pipeline-template/nextflow.config

```diff
@@ -196,7 +196,7 @@ manifest {
     name            = '{{ name }}'
     author          = '{{ author }}'
     homePage        = 'https://github.com/{{ name }}'
-    description     = '{{ description }}'
+    description     = "{{ description }}"
```

Comment:
what happens if one adds double quotes in the description? Not sure what the most python-way to avoid this is, but maybe using `"""` here instead of `"`?